src/Dofs/ConstraintHandler.jl

```diff
@@ -377,6 +404,25 @@ function update!(ch::ConstraintHandler, time::Real=0.0)
         _update!(ch.inhomogeneities, dbc.f, dbc.faces, dbc.field_name, dbc.local_face_dofs, dbc.local_face_dofs_offset,
                  dbc.components, ch.dh, ch.bcvalues[i], ch.dofmapping, ch.dofcoefficients, convert(Float64, time))
     end
+    # Compute effective inhomogeneity for affine constraints with prescribed dofs in the
+    # RHS. For example, in u2 = w3 * u3 + w4 * u4 + b2 we allow e.g. u3 to be prescribed by
+    # a trivial constraint with just an inhomogeneity (e.g. DBC), for example u3 = f(t).
+    # This value have now been computed in _update! and we can compute the effective
+    # inhomogeneity h2 for u2 which becomes h2 = w3 * u3 + b2 = w3 * f3(t) + b2.
+    for i in eachindex(ch.prescribed_dofs, ch.dofcoefficients, ch.inhomogeneities)
```

Comment:
`for (dof, coeffs, h, ac_h) for zip(ch.prescribed_dofs, ch.dofcoefficients, ch.inhomogeneities, ch.affine_inhomogeneities)
` instead ?
(If we make `ch.affine_inhomogeneities` in to a vector instead.)

Reply:
I need the index to insert the result so thought this was cleaner.

Reply:
Ok, I missed that.